### DISCUSSION TITLE
能否增加一个使用移动端api搜索的功能？

### DISCUSSION THREAD

**Discussion by ws2bhhh** [magicdawn/Bilibili-Gate#79] [Ideas]
网页端api搜索结果和移动端api搜索结果不同。希望能增加此类功能，或者能否推荐类似的脚本/项目？
非常感谢。
以及这是可能的实现参考：
[https://github.com/czp3009/bilibili-api/blob/master/src/main/kotlin/com/hiczp/bilibili/api/BilibiliClient.kt](url)

**Comment by magicdawn** [comment]
搜索 out of scope.
可以提给 [BewlyBewly](https://github.com/hakadao/BewlyBewly/)
